### ISSUE TITLE
Is the project still mainained ?

### ISSUE THREAD

**Issue by danielpetisme** [cloudstark/quarkus-zalando-problem-extension#91]
Disclaimer it's neither a bug or a feature request

I'm planning to integrate this extension in JHipster Quarkus.
https://github.com/jhipster/generator-jhipster-quarkus/pull/120

I would like to know the status of the project, do you plan to maintain it?
How can I hep you?

**Comment by SchulteDev** [comment]
I think this project is inactive for time being - @CrystalMethod is the initial committer and project owner, I have not been able to reach him a couple of times and stopped trying getting in contact, as my own motiviation working on this Quarkus extension has vanished meanwhile, too.

**Comment by lenalebt** [comment]
Not that I plan maintaining anything, but I updated dependencies, added support for jackson deserialization and fixed a small bug over at https://github.com/lenalebt/quarkus-zalando-problem-extension. Unsure whether I'll use quarkus in the future; currently only playing around and unsure whether those changes would pull jsonb and jackson into people's dependency trees currently, so nothing to merge yet - but in case anybody wants to play with something slightly updated, there you go :).

**Comment by SchulteDev** [comment]
I think the project is dead. @CrystalMethod is not reachable / does not react for a couple of months meanwhile.

**Comment by lenalebt** [comment]
Yeah, that was more towards other people that might find it. That there is an updated version available. I mean, it's not published anyways to central servers, so - well.

**Comment by lwitkowski** [comment]
Hi guys, this topic is one of the reasons we in TietoEvry decided to create our own implementation of RFC7807 for Quarkus, and today we've finally made it public: https://github.com/TietoEVRY/quarkus-resteasy-problem

It supports both JsonB and Jackson, Native mode, and is compatible with Quarkus 1.4.2+ (1.13.0 as of today), and we've been using this code in production for few months now, so we consider it pretty stable, but we hope the community will make it better :)

**Comment by danielpetisme** [comment]
Hey @lwitkowski, thank for this good news.
I don't know if you're familiar with JHipster (https://www.jhipster.tech/). JHipster is an open-source full-stack application generator, producing both a backend API and a frontend client (Angular, React, etc.).
Since a couple of years, I'm leading the Quarkus implementation 
* https://github.com/jhipster/generator-jhipster-quarkus
* https://www.youtube.com/watch?v=MJe2giXrbuM

We have an open issue about Machine friendly exceptions. This ticket as a $200 bounty
https://github.com/jhipster/generator-jhipster-quarkus/issues/4
If you are interested, feel free to propose an integration of quarkus-resteasy-problem. I would be more than happy to help you.

Again thanks for your work
